bot/commands.ts

```
@@ -1,3 +1,5 @@
+import mongoose from 'mongoose';
+const { ObjectId } = mongoose.Types;
 import { validateFiatSentOrder, validateReleaseOrder } from './validations';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)